client/src/app/pages/migration-waves/migration-waves.tsx

```diff
@@ -400,14 +425,20 @@ export const MigrationWaves: React.FC = () => {
                             </DropdownItem>,
                             <DropdownItem
                               key="bulk-delete"
+                              isDisabled={selectedItems.length === 0}
                               onClick={() => {
+                                let deleteMigrationWaves: Promise<MigrationWave>[] =
+                                  [];
                                 selectedItems.map((migrationWave) => {
```

Comment:
We can use the spread operator here:
```suggestion
                                selectedItems.map((migrationWave) => {
                                  return [
                                    ...deleteMigrationWaves,
                                    migrationWave.id &&
                                      deleteMigrationWave(migrationWave.id),
                                  ];
                                });
```